simulator/ssh-transfer.go

```diff
@@ -0,0 +1,158 @@
+package simulator
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	simssh "github.com/alphasoc/flightsim/simulator/ssh"
+
+	bytesize "github.com/inhies/go-bytesize"
+)
+
+const defaultSendSize = 100 * bytesize.MB
+
+// TODO where will the server side live?
+var defaultTargetHosts = []string{"127.0.0.1:22", "127.0.0.1:9999"}
```

Comment:
this should point to `sandbox.alphasoc.xyz` by default

Reply:
Great, thanks.